src/v40/EntryPoint40.js

```diff
@@ -27,26 +27,34 @@ const { ToucheggConfig } = SRC.touchegg.ToucheggConfig;
 const { AllowedGesture } = SRC.utils.AllowedGesture;
 const { logger } = SRC.utils.Logger;
 
+const ExtensionUtils = imports.misc.extensionUtils;
+const Me = ExtensionUtils.getCurrentExtension();
+const Convenience = Me.imports.convenience;
+
 /**
  * Entry point for GNOME Shell 40.
  */
 class EntryPoint40Class extends GObject.Object {
   static start() {
+    let settings = Convenience.getSettings('org.gnome.shell.extensions.x11gestures');
+    let fingers = settings.get_int('swipe-fingers');
+    let cfg = {fingers};
+
     const allowedGestures = [
-      EntryPoint40Class.hookGlobalSwitchDesktop(),
-      EntryPoint40Class.hookGlobalOverview(),
-      EntryPoint40Class.hookActivitiesSwitchDesktop(),
+      EntryPoint40Class.hookGlobalSwitchDesktop(cfg),
+      EntryPoint40Class.hookGlobalOverview(cfg),
+      EntryPoint40Class.hookActivitiesSwitchDesktop(cfg),
     ];
 
     ToucheggConfig.update(allowedGestures);
   }
 
-  static hookGlobalSwitchDesktop() {
+  static hookGlobalSwitchDesktop({fingers}) {
```

Comment:
No need to pass this as an object, but the linter complains:

```suggestion
  static hookGlobalSwitchDesktop({ fingers }) {
```